### PR TITLE
Ports the Paramedic Update from upstream

### PR DESCRIPTION
**Pull request by Michiyamenotehifunana**
## About The Pull Request

Paramedics now get an armor that can buff their speed for 30 seconds every 5 minutes, also they get a cool helmet. They also get a way cooler tablet that points to where your selected track is, and also they get a gun.

Also suit sensor monitoring interface has been updated.

## Why It's Good For The Game

discount trauma team

## Changelog
```changelog
add: Added tons of new stuff for paramedics including the tracking tablet and speedy armor
```

### CODE REVIEW

**Comment by bearrrrrrrr** [comment]
pog

**Comment by Jamini** [comment]
Guns go in lockers, and I thought this was explicitly voted against last time?

File: zzzz_modular_occulus/code/datums/outfits/jobs/medical.dm [hunk 1]
```diff
@@ -0,0 +1,4 @@
+//modular override so paramedics don't spawn with guns in their bag
+
+/decl/hierarchy/outfit/job/medical/paramedic
+	backpack_contents = list(/obj/item/modular_computer/tablet/moebius/preset = 1)
```
Comment (Michiyamenotehifunana) [inline zzzz_modular_occulus/code/datums/outfits/jobs/medical.dm:4]: On-spawn pew pews removed here

**Review by Jamini — APPROVED** [review]
I'm willing to give it a shot. Tori does do the best PRs even if I give him all the shit.

**Pull request merged**